pkg/api/ldap_debug.go

```diff
@@ -98,7 +100,7 @@ func (server *HTTPServer) ReloadLDAPCfg() Response {
 
 	err := ldap.ReloadConfig()
 	if err != nil {
-		return Error(http.StatusInternalServerError, "Failed to reload ldap config.", err)
+		return Error(http.StatusInternalServerError, "Failed to reload ldap config", err)
```

Comment:
Since you are here I would change the name from `ldap` -> `LDAP`, I believe everywhere else we use the official abbreviation style